tests/v2/validation/rbac/rbac_test.go

```diff
@@ -273,39 +267,17 @@ func (rb *RBTestSuite) ValidateDeleteProject(role string) {
 
 func (rb *RBTestSuite) ValidateRemoveClusterRoles() {
 
-	log.Info("Removing added cluster member from the cluster as an admin")
 	err := users.RemoveClusterRoleFromUser(rb.client, rb.standardUser)
 	require.NoError(rb.T(), err)
 }
 
 func (rb *RBTestSuite) ValidateRemoveProjectRoles() {
 
-	log.Info("Removing added project member from the cluster projects as an admin")
 	err := users.RemoveProjectMember(rb.client, rb.standardUser)
 	require.NoError(rb.T(), err)
 
 }
 
-func (rb *RBTestSuite) ValidateListGlobalSettings() {
```

Comment:
So Anu we're removing these validations as well?

Reply:
This was for additional tests from rbac which was not removed in the previous PRs. Removing them in this PR.

Reply:
Ok cool.